pkg/controllers/management/auth/project_cluster_handler.go

```diff
@@ -221,16 +227,21 @@ func (l *clusterLifecycle) sync(key string, orig *apisv3.Cluster) (runtime.Objec
 	return nil, nil
 }
 
+// Create creates a Project cluster.
```

Comment:
Technically, this function does nothing because the create is handled by `sync`.